src/modules/UI/scenes/SendConfirmation/SendConfirmation.ui.js

```diff
@@ -327,7 +329,12 @@ export class SendConfirmation extends Component<Props, State> {
       const fiatFeeSymbol = secondaryInfo.displayDenomination.symbol ? secondaryInfo.displayDenomination.symbol : ''
       const exchangeConvertor = convertNativeToExchange(primaryInfo.exchangeDenomination.multiplier)
       const cryptoFeeExchangeAmount = exchangeConvertor(networkFee)
-      const fiatFeeAmount = this.props.currencyConverter.convertCurrency(this.props.currencyCode, secondaryInfo.exchangeCurrencyCode, cryptoFeeExchangeAmount)
+      const fiatFeeAmount = convertCurrencyFromExchangeRates(
+        exchangeRates,
```

Comment:
Looks like mostly formatting change and using our new `convertCurrencyFromExchangeRates`